### PR TITLE
Add Unique URLs to Appointment Management Page

### PR DESCRIPTION
**Pull request by MatthewMeacham**
Just added the self-service appointment reschedule URL and the upload documents URL to the appointment management page so that a volunteer could fetch these for a client if the client didn't save the information.

### CODE REVIEW

File: server/management/appointments/appointments.php [hunk 1]
```diff
@@ -49,7 +49,7 @@ function getAppointments($year) {
 				JOIN AppointmentTime ON Appointment.appointmentTimeId = AppointmentTime.appointmentTimeId
 				JOIN Site ON AppointmentTime.siteId = Site.siteId
 				JOIN Client ON Appointment.clientId = Client.clientId
-				JOIN SelfServiceAppointmentRescheduleToken ON Appointment.appointmentId = SelfServiceAppointmentRescheduleToken.appointmentId
+				LEFT JOIN SelfServiceAppointmentRescheduleToken ON Appointment.appointmentId = SelfServiceAppointmentRescheduleToken.appointmentId
```
Comment (jcarrigan2) [inline server/management/appointments/appointments.php:52]: What's the reason for this change? Thanks~

Comment (MatthewMeacham) [reply server/management/appointments/appointments.php:52]: I needed the `token` in order to construct the unique URLs. The left join is because not every appointment has a token (in particular, the appointments that were created before the token infrastructure was built or ones that were created manually by us rather than through the site), but we still want all the appointments.
